nanoqm/common.py

```diff
@@ -42,6 +42,20 @@
 from qmflows.type_hints import PathLike
 from scm.plams import Atom, Molecule
 
+# TODO: Move this `try`/`accept` import after some time
+try:
+    from qmflows.yaml_utils import UniqueSafeLoader as UniqueSafeLoader
+except ImportError as ex:
+    import warnings
+    from yaml import SafeLoader as UniqueSafeLoader  # type: ignore[misc]
+
+    _warning = ImportWarning("Failed to import `qmflows.yaml_utils.UniqueSafeLoader`;"
+                             " falling back to `yaml.SafeLoader`")
+    _warning.__cause__ = ex
+
+    warnings.warn(_warning)
+    del _warning, warnings
+
```

Comment:
In principle we could also import without the `try`/`except` clause, but that will introduce a hard dependency on the latest qmflows version that was merged a few minutes ago.

Any thoughts?

Reply:
Having a hard dependency on the latest QMFlows is not a  big issue because the two libraries are highly coupled.